cpp/src/strings/convert/convert_urls.cu

```diff
@@ -328,82 +382,54 @@ std::unique_ptr<column> url_decode(
   size_type strings_count = strings.size();
   if (strings_count == 0) return make_empty_column(data_type{type_id::STRING});
 
+  constexpr int threadblock_size      = 128;
+  constexpr int block_size            = 256;
+  constexpr int warps_per_threadblock = threadblock_size / cudf::detail::warp_size;
```

Comment:
As above, maybe best to define this quantity and make `threadblock_size` the derived quantity?

Reply:
Addressed in https://github.com/rapidsai/cudf/pull/8622/commits/961ed39a28b0bcac1f2d4abcc2b6855309535843.